symphony-bdk-spring/bdk-app-starter/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
should it be used by the example then?

Reply:
Just created the module, no implementation provided yet